src/OpenSage.Game/Gui/Wnd/Controls/Label.cs

```diff
@@ -11,6 +11,13 @@ public Label(WndWindowDefinition wndWindow)
             TextAlignment = wndWindow.StaticTextData.Centered
                 ? TextAlignment.Center
                 : TextAlignment.Leading;
+
+            if(wndWindow.EnabledDrawData.Items.Length > 0)
+            {
+                BorderColor = wndWindow.EnabledDrawData.Items[0].BorderColor.ToColorRgbaF();
+            }
+            
+            BorderWidth = 1;
```

Comment:
i believe this is default behavior for when the control doesn't have a background image set.
https://github.com/OpenSAGE/OpenSAGE/blob/73f61bb3fef7dc9eee2f3f4fcfc48cee37190cad/src/OpenSage.Game/Gui/Wnd/Controls/Control.Wnd.cs#L115-L130

the only difference is that labels don't have a background image. should we even support that?
i don't remember seeing any labels with images in the game though

i've also noticed that above code loads a BackgroundColor, i don't know if labels should care about it as well

Reply:
To me _ideally_ we support whatever is possible in the wnd files. It's sort of looking a bit ahead, but ZH modders do some weird stuff with the wnd files and it would be nice to be able to accommodate that, even if those engine features were never used in the base game itself.